cirq/optimizers/decompositions_test.py

```diff
@@ -31,6 +32,29 @@ def assert_gates_implement_unitary(gates: Sequence[cirq.SingleQubitGate],
                                                     atol=atol)
 
 
+def test_is_negligible_turn():
+    assert cirq.is_negligible_turn(0, 1e-5)
+    assert cirq.is_negligible_turn(1e-6, 1e-5)
+    assert cirq.is_negligible_turn(1, 1e-5)
+    assert cirq.is_negligible_turn(1 + 1e-6, 1e-5)
+    assert cirq.is_negligible_turn(1 - 1e-6, 1e-5)
+    assert cirq.is_negligible_turn(-1, 1e-5)
+    assert cirq.is_negligible_turn(-1 + 1e-6, 1e-5)
+    assert cirq.is_negligible_turn(-1 - 1e-6, 1e-5)
+    assert cirq.is_negligible_turn(3, 1e-5)
+    assert cirq.is_negligible_turn(3 + 1e-6, 1e-5)
+    assert not cirq.is_negligible_turn(1e-4, 1e-5)
+    assert not cirq.is_negligible_turn(-1e-4, 1e-5)
+    assert not cirq.is_negligible_turn(0.5, 1e-5)
+    assert not cirq.is_negligible_turn(-0.5, 1e-5)
+    assert not cirq.is_negligible_turn(0.5, 1e-5)
+    assert not cirq.is_negligible_turn(4.5, 1e-5)
+    assert not cirq.is_negligible_turn(sympy.Symbol('a'), 1e-5)
+    # Sympy expression for a constant
+    assert cirq.is_negligible_turn(sympy.Symbol('a') * 0 + 3 + 1e-6, 1e-5)
```

Comment:
it feels strange to me that this is true, since 'a' can take on some value which is not near an integer.

Reply:
a is multiplied by zero so it doesn't matter.

Reply:
could you add the case where a symbol appears in a formula, not just alone 
`assert not cirq.is_negligible_turn(sympy.Symbol('a') + 1 - 1e-6, 1e-5)`

Reply:
Done.